cmd/utils/flags.go

```diff
@@ -1718,6 +1740,25 @@ func setMiner(ctx *cli.Context, cfg *miner.Config) {
 	if ctx.IsSet(RollupComputePendingBlock.Name) {
 		cfg.RollupComputePendingBlock = ctx.Bool(RollupComputePendingBlock.Name)
 	}
+	if ctx.IsSet(MevEnabledFlag.Name) {
+		cfg.Mev.MevEnabled = ctx.Bool(MevEnabledFlag.Name)
+	}
+	if ctx.IsSet(SentryEnabledFlag.Name) {
+		cfg.Mev.SentryEnabled = ctx.Bool(SentryEnabledFlag.Name)
+	}
+	if ctx.IsSet(SequencerUrl.Name) {
+		url := ctx.String(SequencerUrl.Name)
+		if strings.Contains(url, ",") {
+			cfg.Mev.Sequencers = strings.Split(url, ",")
+		} else {
```

Comment:
There is no need to add `if else` here. `strings.Split(url, ",")` works for both branch.

Reply:
Fixed, please review again.